src/main/java/seedu/address/logic/Messages.java

```diff
@@ -21,6 +21,7 @@ public class Messages {
                 "Multiple values specified for the following single-valued field(s): ";
     public static final String MESSAGE_INVALID_EVENT_DISPLAYED_INDEX = "The event index provided is invalid";
     public static final String MESSAGE_SELECT_EVENT = "No event is selected!";
+    public static final String MESSAGE_INVALID_DELETE_EVENT = "No event can be deleted when it is selected";
```

Comment:
Put this inside DeleteCommand.java, as it is specific to DeleteCommand.
Change it to "The selected Event cannot be deleted"